src/play_cljc/gl/core.cljc

```diff
@@ -94,7 +94,7 @@
                       (gl game bindFramebuffer (gl game FRAMEBUFFER) previous-framebuffer)
                       fb))}))
 
-(defn- call-uniform* [game m glsl-type uni-loc uni-name data]
+(defn- call-uniform* [game m glsl-type ^Integer uni-loc uni-name data]
```

Comment:
After testing them all individually, this is the only type-hint that makes a clearly visible difference on its own